app/models/iiif_presentation_v3.rb

```diff
@@ -72,6 +73,20 @@ def manifest_descriptive_properties
     @manifest["metadata"] = metadata
   end
 
+  # rubocop:disable Rails/TimeZone
+  def manifest_navdate
+    date = @parent_object.authoritative_json['dateStructured'] if @parent_object&.authoritative_json
+    if date.nil?
+      @manifest['navDate'] = nil
+    elsif date&.first&.include?('/')
+      first_year = date.first.split('/').first
+      @manifest['navDate'] = DateTime.new(first_year.to_i)
```

Comment:
Force the string conversion with `Z` instead of `+00:00` as follows:
DateTime.new(first_year.to_i).utc.iso8601